src/eva/core/data/datasets/base.py

```diff
@@ -26,6 +26,10 @@ def setup(self) -> None:
         """
         self.configure()
         self.validate()
+        self._setup()
+
+    def _setup(self) -> None:
```

Comment:
why we have this method? for setups we should use the `self.configure()` method, or is this a blocker?

Reply:
i see, @nkaenzig do you see anything that speaks against removing `_setup` and replacing it with `configure` in `MultiWsiDataset`?

Reply:
I think using configure is fine, I had forgotten this existed :D

Reply:
move validate to configure